e2e/various-suite/loki-table-explore-to-dash.spec.ts

```diff
@@ -110,7 +110,7 @@ const lokiQueryResult = {
   },
 };
 
-describe('Loki Query Editor', () => {
+describe.skip('Loki Query Editor', () => {
```

Comment:
don't forget to tag someone in slack to unskip this 👍

Reply:
Thanks for reminder 👍